testing-modules/selenium-2/src/test/java/com/baeldung/selenium/alertsandpopups/HandlePopupsLiveTests.java

```diff
@@ -0,0 +1,63 @@
+package com.baeldung.selenium.alertsandpopups;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.time.Duration;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class HandlePopupsLiveTests {
```

Comment:
The naming convention for tests can be found here: https://github.com/eugenp/tutorials/blob/e759c08ec8a18c35ef15294800342074cba70a39/custom-pmd/src/main/java/com/baeldung/pmd/UnitTestNamingConventionRule.java#L17
```suggestion
public class HandlePopupsLiveTest {
```